.travis.yml

```diff
@@ -5,15 +5,14 @@ language: python
 
 dist: trusty
```

Comment:
Drop trusty because it is EOL.  https://blog.travis-ci.com/2019-04-15-xenial-default-build-environment